distro/src/avail/Avail.avail/Foundation.avail/Tuples.avail

```diff
@@ -3243,18 +3344,36 @@ Semantic restriction "_'s⁇last" is
  */
 Public stable method "∑_" is
 [
-	intTuple : extended integer*
+	intTuple : extended integer+
```

Comment:
This should probably continue to work for empty tuples, using zero as the natural base accumulator value.